src/components/dashboard/main-content/course-enrollments/tests/CourseEnrollments.test.jsx

```diff
@@ -100,6 +145,21 @@ describe('Course enrollments', () => {
     expect(screen.getAllByText(inProgCourseRun.title).length).toBeGreaterThanOrEqual(1);
   });
 
+  it('does not render cancelled assignment and renders cancelled alert', async () => {
+    renderEnrollmentsComponent();
+    expect(screen.getByText('Course assignment cancelled')).toBeInTheDocument();
+    expect(screen.queryByText('test-title')).toBeFalsy();
+    const dismissButton = screen.getByText('Dismiss');
+    await act(async () => userEvent.click(dismissButton));
```

Comment:
`userEvent` methods shouldn't need to be wrapped in `act`; they are already wrapped with `act` under the hood.